kano_updater/ui/progress.py

```diff
@@ -24,10 +24,12 @@ def _change(self, phase, msg):
                       phase.get_main_phase().label, phase.get_main_phase().name, msg)
 
     def _error(self, phase, msg):
+        kill_flappy_judoka()
```

Comment:
I added `kill_flappy_judoka` in quite a few other places. Is this the only redundant one?